tools/topology/topology2/include/pipelines/cavs/mixout-playback.conf

```diff
@@ -56,15 +56,23 @@ Class.Pipeline."mixout-playback" {
 			node_type $HDA_LINK_OUTPUT_CLASS
 			period_sink_count 0
 			period_source_count 2
-			num_audio_formats 2
+			num_audio_formats 3
 			# 16-bit 48KHz 2ch
 			Object.Base.audio_format.1 {
 				in_bit_depth		32
 				in_valid_bit_depth	32
 				dma_buffer_size "$[$obs * 2]"
 			}
-			# 32-bit 48KHz 2ch
+			# 24-bit 48KHz 2ch
 			Object.Base.audio_format.2 {
+				in_bit_depth		32
+				in_valid_bit_depth	32
+				out_bit_depth		32
+				out_valid_bit_depth	24
+				dma_buffer_size "$[$obs * 2]"
+			}
+			# 32-bit 48KHz 2ch
+			Object.Base.audio_format.3 {
```

Comment:
isn't this going to conflict with @ranj063's changes?

Reply:
> isn't this going to conflict with @ranj063's changes?

Right, updated.